data-prepper-plugins/mongodb/src/main/java/org/opensearch/dataprepper/plugins/mongo/model/CheckpointStatus.java

```diff
@@ -0,0 +1,45 @@
+package org.opensearch.dataprepper.plugins.mongo.model;
+
+public class CheckpointStatus {
```

Comment:
This class is only used by `StreamAcknowledgementManager`. So you can move this into the `stream` package and make it package-private to help keep it encapsulated.

Reply:
Sure. will do.